packages/attestation-service/src/models/attestation.ts

```diff
@@ -66,16 +66,16 @@ export default (sequelize: Sequelize) => {
     completedAt: DataTypes.DATE,
   }) as AttestationStatic
 
-  model.prototype.key = function(): AttestationKey {
+  model.prototype.key = function (): AttestationKey {
```

Comment:
This seems like such a minor thing, maybe better off sticking with the current setup?

Can we configure this in our prettier.config.js file?

Reply:
I don't actually think there's an option to change this (will update the comment above -- there are only **some** options that we could overwrite to get the "old style"). They motivate this change [here](https://prettier.io/blog/2020/03/21/2.0.0.html#javascript-1) as moving towards consistency in spacing around the `function` keyword.

Reply:
After reading (parts of) [this thread](https://github.com/prettier/prettier/pull/3903) I agree with shipping this change.

We should be consistent with Prettier even if it does mean a minor inconvenience.

Reply:
Hey @eelanagaraj, I searched for `function(` and I noticed that in our docs we have not adopted the new prettier style, would we want to do this?

These are the instances
```
packages/docs/developer-resources/walkthroughs/hello-contract-remote-node.md
83:module.exports = function(deployer) {

packages/docs/developer-resources/walkthroughs/hellocontracts.md
52:module.exports = function(deployer) {

packages/docs/developer-resources/dappkit/setup.md
85:  global.btoa = function(str) {

```

Reply:
Ooh interesting @piersy, will do so for the `function` style change in particular.

Probably a good idea to add auto-formatting to our docs style anyways (since I'm pretty sure it's quite inconsistent at the moment), maybe something we can look into later cc @aslawson @barbaraliau? I think it's out of scope for this particular PR though.